### PR TITLE
Basic fetch functionality

### PR DESCRIPTION
**Pull request by wojw5**
https://trello.com/c/P4XCgbPH

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,47 @@ Or install it yourself as:
...
+
+```ruby
+product: :research
+data_category: :links
+method: :get
+return_data: :list
+params: { }
+```
+
+TODO: Write more usage instructions here
```
Comment (saschaknobloch) [inline README.md]: Could you remove the `TODO` and add how the data structure looks like which is returned by the call you are showing?

File: lib/searchmetrics_client/request.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    attr_reader :url, :query, :response
+
+    extend Forwardable
+    def_delegator :query, :type
+
+    def self.send_request(query)
+      new(query).send_request
+    end
+
+    def self.send_request_form_hash(hash)
```
Comment (saschaknobloch) [inline lib/searchmetrics_client/request.rb]: Is this a typo? ^^

File: lib/searchmetrics_client/query.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
+module SearchmetricsClient
+  class Query
+    attr_accessor :query, :product, :data_category, :method, :return_data, :type, :params
+
+    def self.from_hash(hash)
+      product       = hash.fetch(:product, :research)
+      data_category = hash.fetch(:data_category, :links)
+      method        = hash.fetch(:method, :get)
+      return_data   = hash.fetch(:return_data, :list)
```
Comment (saschaknobloch) [inline lib/searchmetrics_client/query.rb]: Why did you decide to put exactly those values as defaults?

Comment (wojw5) [inline lib/searchmetrics_client/query.rb]: I looks like those values occur most frequently in api documentation.

**Pull request merged**